### ISSUE TITLE
[BUG] Alt text field for images not visible with white images

### ISSUE THREAD

**Issue by pgundlach** [mastodon/mastodon-ios#863]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I am using the mastodon client on macOS X (Ventura 13.1) and try to add an alt text for an image. On a white image, I cannot see where to type the alt text. See the following example

<img width="551" alt="compose" src="https://user-images.githubusercontent.com/209434/211024972-7773a135-84e4-4a5d-b2dc-196e3d04d5ec.png">

When the image has a white background (not coloured like here), you cannot see what you are typing.

### Expected Behavior

I'd rather have the alt text below or above the image or an edit button for the alt text.

### Environment

```markdown
- Device: macbook
- OS: Ventura (13.1)
- Version: Version: 1.4.9
- Build: 224
```

**Comment by j-f1** [comment]
Pretty sure this was just fixed by #800!

**Comment by pgundlach** [comment]
I'll close the issue. I swear, I have used the search before ;-)
If I still have problems after the release, I'll re-open this with more information.

Thank you very much.
